package.json

```diff
@@ -9,10 +9,10 @@
     "url": "git://github.com/localtunnel/localtunnel.git"
   },
   "dependencies": {
-    "request": "2.81.0",
-    "yargs": "3.29.0",
+    "axios": "^0.17.1",
```

Comment:
Please follow existing dependency convention and remove the `^`. When contributing to projects always try to make your changes look like the nearby code/configuration style.

Reply:
fixed via 80198e1